geonode/geoserver/tasks.py

```diff
@@ -179,6 +180,9 @@ def geoserver_create_style(
                     get_sld_for(gs_catalog, instance)
             else:
                 get_sld_for(gs_catalog, instance)
+            if not f:
+                # trigger signal to set geoserver style visual_mode automatically
+                geoserver_style_visual_mode.send_robust(sender=instance, instance=instance)
```

Comment:
@marthamareal the signal is generic, so it's name says what is happened, not what should be done :)
Please rename it to `geoserver_automatic_default_style_set`, and change the comment in "this signal is used by the mapstore client to set the style in visual mode"

Reply:
@giohappy updated.